OpenRA.Game/Translation.cs

```diff
@@ -51,10 +54,10 @@ public Translation(string language, string[] translations, IReadOnlyFileSystem f
 				.UseConcurrent()
 				.UncheckedBuild();
 
-			ParseTranslations(language, translations, fileSystem);
+			ParseTranslations(language, translations, fileSystem, onError);
 		}
 
-		void ParseTranslations(string language, string[] translations, IReadOnlyFileSystem fileSystem)
+		void ParseTranslations(string language, string[] translations, IReadOnlyFileSystem fileSystem, Action<Linguini.Syntax.Parser.Error.ParseError> onError)
```

Comment:
Use a using instead of specifying the full type inside `Action<>`?

Reply:
No I mean why are you typing `Linguini.Syntax.Parser.Error.ParseError` instead of declaring a `using` and using the simple name.

Reply:
Ah I misread completely. That makes sense